README.md

```diff
@@ -156,6 +169,8 @@ RNeXML:::nexml_namespaces
 ##               "http://purl.org/dc/elements/1.1/" 
 ##                                          dcterms 
 ##                      "http://purl.org/dc/terms/" 
+##                                              ter 
+##                      "http://purl.org/dc/terms/" 
```

Comment:
So why is there now a second occurrence (with a different prefix) of the `http://purl.org/dc/terms/` namespace? That seems indicative of a bug somewhere.

Reply:
It's there because the URI is defined in the default `nexml_namespaces` list under two different prefixes, for whatever reason.